packages/contentstack-bootstrap/src/commands/cm/bootstrap.ts

```diff
@@ -152,6 +152,10 @@ export default class BootstrapCommand extends Command {
       const yes = bootstrapCommandFlags.yes as string;
 
       const appConfig: AppConfig = getAppLevelConfigByName(selectedAppName || selectedApp.configKey);
+      let master_locale = "en-us";
+      if( selectedApp.configKey == "compass-app" || selectedAppName == "Compass App"){
```

Comment:
can't we use this from the config, rather than adding a condition for compass app here, appConfig.masterLocale || "en-us"

  'compass-app': {
      source: 'SunilLsagar/universal-demo',
      stack: 'SunilLsagar/stack-universal-demo',
      master_locale: 'en',
    },